### PR TITLE
DHFPROD-7735: Component Replacement: Divider

### PR DESCRIPTION
**Pull request by MSBrignoni**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by aniamy — APPROVED** [review]

**Review by iveyMarklogic — APPROVED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@MSBrignoni there are some lint UI failures on the PR, Please fix them

**Review by wooldridge — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/src/components/common/divider/divider.module.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.color {
+	color:#a1a1a1;
```
[the remark this answers is not in the document]
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/components/common/divider/divider.module.scss]: Resolved!.

File: marklogic-data-hub-central/ui/src/components/job-response/job-response.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React, {useState, useEffect, useContext} from "react";
-import {Descriptions, Divider, Modal, Icon, Collapse} from "antd";
+import {Descriptions, Modal, Icon, Collapse, Spin, Button} from "antd";
```
[the remark this answers is not in the document]
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/components/job-response/job-response.tsx]: Resolved.

File: marklogic-data-hub-central/ui/src/components/common/divider/divider.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import styles from "./divider.module.scss";
+
+interface Props {
+  type?: "horizontal" | "vertical";
+  className?: string;
+  dashed?: boolean;
+  style?: React.CSSProperties;
+}
+
+const Divider: React.FC<Props> = (props) => {
```
[the remark this answers is not in the document]
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/components/common/divider/divider.tsx]: Resolved.

**Review by iveyMarklogic — APPROVED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Review by aniamy — APPROVED** [review]

**Review by aniamy — APPROVED** [review]

**Review by wooldridge — APPROVED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Review by mrodrigu-marklogic — APPROVED** [review]

**Pull request merged**